controllers/twophase/types.go

```diff
@@ -92,15 +92,50 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 		status.Experiment.Phase = v1alpha1.ExperimentPhaseFinished
 		status.FailedMessage = emptyString
-	} else if chaos.IsPaused() {
+	} else if chaos.GetPause() != "" {
 		if status.Experiment.Phase == v1alpha1.ExperimentPhaseRunning {
 			r.Log.Info("Pausing")
 
-			err = r.Recover(ctx, req, chaos)
-			if err != nil {
-				r.Log.Error(err, "failed to pause chaos")
-				updateFailedMessage(ctx, r, chaos, err.Error())
-				return ctrl.Result{Requeue: true}, err
+			if chaos.GetPause() == "true" {
+				err = r.Recover(ctx, req, chaos)
+				if err != nil {
+					r.Log.Error(err, "failed to pause chaos")
+					updateFailedMessage(ctx, r, chaos, err.Error())
+					return ctrl.Result{Requeue: true}, err
+				}
+			} else {
+				// Pause time is set
+				var pauseSpec v1alpha1.SchedulerSpec
```

Comment:
I think `SchedulerSpec` is not necessary here, because we can only use `time.Now().Add(duration)` to  implement this.

Reply:
Thanks! Didn't know time could parse string to duration by then. fixed~